### ISSUE TITLE
Hell of the cauldrons of oil is broken

### ISSUE THREAD

**Issue by McFrugal** [immortalityidle/immortalityidle.github.io#195]
If you bring in over 1000 ice cores, the progress meter is full and you get reduced to 0 hp from the steam... but nothing happens.

**Comment by Enamahc** [comment]
Did you manage to complete this hell? I have the same issue. Is it a bug or is there something i'm missing?

**Comment by McFrugal** [comment]
Yes, the problem is you need to do it and also have enough hp to endure the damage without dropping to 0.
